cmd/cadvisor.go

```diff
@@ -145,6 +153,10 @@ func main() {
 	defer klog.Flush()
 	flag.Parse()
 
+	if os.Getenv("ZIPKIN") != "" {
```

Comment:
Please use flags instead of env vars for configuration.  This applies here and elsewhere.

Reply:
I will refactore